cocos/editor-support/cocostudio/ActionTimeline/CCFrame.cpp

```diff
@@ -149,7 +149,8 @@ TextureFrame* TextureFrame::create()
 }
 
 TextureFrame::TextureFrame()
-    : _textureName("")
+    : _sprite(nullptr)
+    , _textureName("")
```

Comment:
Is the modification related to the issue?

Reply:
The following two modifications is not related to this issue, it crashed accidentally. I found it because of the problems caused by the constructor, some varibles have not been initialized.